docs/reference/Signed-Packages-Reference.md

```diff
@@ -18,7 +18,7 @@ Signed packages provide the strongest end-to-end validation. There are two diffe
 - **Author signature**. An author signature guarantees that the package has not been modified since the author signed the package, no matter from which repository or what transport method the package is delivered. Additionally, author-signed packages provide an extra authentication mechanism to the nuget.org publishing pipeline because the signing certificate must be registered ahead of time. For more information, see [Register certificates](#signature-requirements-on-nugetorg).
 - **Repository signature**. Repository signatures provide an integrity guarantee for **all** packages in a repository whether they are author signed or not, even if those packages are obtained from a different location than the original repository where they were signed.   
 
-For details on creating an author signed package, see [Signing Packages](../create-packages/Sign-a-package.md) and the [nuget sign command](../reference/cli-reference/cli-ref-sign.md). You can verify packages' signatures using the [dotnet nuget verify](/dotnet/core/tools/dotnet-nuget-verify.md) or [nuget verify](../reference/cli-reference/cli-ref-verify.md) commands.
+For details on creating an author signed package, see [Signing Packages](../create-packages/Sign-a-package.md) and the [nuget sign command](../reference/cli-reference/cli-ref-sign.md). You can verify packages' signatures using the [dotnet nuget verify](/dotnet/core/tools/dotnet-nuget-verify) or [nuget verify](../reference/cli-reference/cli-ref-verify.md) commands.
 
```

Comment:
Do we need to do same thing for `../reference/cli-reference/cli-ref-verify.md`? Maybe for you no need to do any changes.

Reply:
`nuget.exe verify` is under the same root of https://docs.microsoft.com/en-us/nuget/ but `dotnet nuget verify` command is in `dotnet/docs` repo https://docs.microsoft.com/en-us/dotnet/. It is not required to modify links for `nuget.exe` verify because it is not broken.

Reply:
Browse https://docs.microsoft.com/en-us/nuget/reference/signed-packages-reference and click on `nuget verify` (link is not broken) whereas click on `dotnet nuget verify` (links is broken).